src/sentry/digests/backends/redis.py

```diff
@@ -1,13 +1,16 @@
 import logging
 import time
 from contextlib import contextmanager
+from typing import Any, Iterable, Optional, Tuple
 
-from redis.client import ResponseError
+from rb.clients import LocalClient  # type: ignore
+from redis.client import ResponseError  # type: ignore
```

Comment:
Is this because we don't have type stubs for these libraries? Can we configure this in mypy.ini?

Reply:
Just checked (the docs)[https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-type-hints-for-third-party-library] and they says we can find or create type stubs, ignore once like I have here, or add
```
[mypy]
ignore_missing_imports = True
``
to always ignore this class of type errors.

Reply:
Yes, I think either of those is better than ignoring on every import